src/RouterConfig.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Navbar, Nav, NavItem, NavDropdown, MenuItem, FormGroup, Form, FormControl, Grid, Row, Col, Button } from 'react-bootstrap'
+import { Link, Switch, Redirect, Route, BrowserRouter } from 'react-router-dom'
+import { LinkContainer } from 'react-router-bootstrap'
+import './App.css';
+import 'bootstrap/dist/css/bootstrap.min.css'  // temporary, we can create our css later
+import AppointmentHome from './Appointment/AppointmentHome'
+import ServiceHome from './Service/ServiceHome'
+import RequestHome from './Request/index'
```

Comment:
`'./Request/index'` export a component called `index`, not `RequestHome`

Reply:
In my part, I made index file to call both RequestHome and  SideBar so I called it. However, It makes some errors when I call another page under Request directory, not defined in index, It does not contain SideBar in the render. 

I need to think it more .